### ISSUE TITLE
make a spreadsheet editor

### ISSUE THREAD

**Issue by mirlip** [mauge123/mechanical-blender#23]
Hi Mauge,
Again some time to test your awesome branch :) Would it be possible to make a spreadsheet editor? Many cases in CAD need some statistics, custom informations, etc... Being able to edit and organise them, to also use them as selection conditions would be really helpful. I don't know how hard it is to create a new editor with vertical and horizontal lines. Editable fields are already implemented and maybe reusable?
Regards

**Comment by mauge123** [comment]
Yes a spreadsheet editor will be useful, but doing a new editor is not easy, i'll try some time ago, and will do to get formal drawings. 

What do you mean as using it as selection conditions ?

**Comment by mirlip** [comment]
I mean, if you click on a row, it select the row of course, but also the element it references in the 3D view (a face, an object, a mesh, whatever it is)
